get-latest-tgf.sh

```diff
@@ -1,20 +1,38 @@
 #!/usr/bin/env bash
 
-TGF=${TGF_PATH:=/usr/local/bin}/tgf
+DEFAULT_INSTALL_DIR=/usr/local/bin
+TGF=${TGF_PATH:=${DEFAULT_INSTALL_DIR}}/tgf
 
-if [ ! -d "$TGF_PATH" ]; then
-  echo "creating "$TGF_PATH" directory..."
-  mkdir -p $TGF_PATH
+if [ ! -d "${TGF_PATH}" ]; then
+  echo "creating ${TGF_PATH} directory..."
+  mkdir -p ${TGF_PATH}
+fi
+
+if [ ! -w "${TGF_PATH}" ]; then
+  if [ ${TGF_PATH} == "${DEFAULT_INSTALL_DIR}" ]; then
+    # This is a system directory.  It's not a good idea to try to make it writeable.
+    echo "System installation directory ${TGF_PATH} is not writeable."
+    echo "Please set the TGF_PATH environment variable to install to an alternate, writeable directory."
```

Comment:
Should we tell users to add sudo if they really want to install there. That's what I do

Reply:
I prefer not.  It means we have to update the README to explain this and show where the sudo must go (curl | sudo bash), not (sudo curl | bash, a common mistake).  And, we'd also have to warn that "tgf --update" will need sudo to work, so we'd have to change the tgf user's guide too, or maybe add logic in tgf --update to detect this and tell the user to run it with sudo.

I don't think it's worth adding all that given that tgf doesn't really need any sudo permissions to work.